### ISSUE TITLE
stis idctable

### ISSUE THREAD

**Issue by SaOgaz** [spacetelescope/stwcs#56]
I've been told by the STIS team that updates they have made to the idctable file are not being properly accounted for.  We are having a meeting today to discuss this with their team.
@stscicrawford @stscirij @stsci-hack

**Comment by SaOgaz** [comment]
Notes from meeting today:

There are various concerns going on for both how CALSTIS is applying the IDCTAB in addition to how it's being handled in stwcs. If there are incorrect calculations going on, they are likely to be small ones.  Issues are related to the STIS IDCTAB file being formatted differently than the other HST instruments, and CALSTIS applying a reverse of the inverse correction stored in the IDC table, as oppose to apply a forward correction.

In addition, when Tony attempted to run astrodrizzle with one of their updated IDCTAB files, it crashed.  He stated that he needs to double check that he was running astrodrizzle correctly, and was not 100% confident that the crash was due to the new IDC Table file.

Our current regression tests are not using the updated IDCTAB file as the datasets were pulled from MAST too long ago.  @stsci-hack is going to follow up on this.

With regards to the correct (or incorrect) behavior of stwcs and CALSTIS, the STIS team is going to continue to investigate and pin down the problem.  In addition they will be making a summary of the time commitment from their side for fixing any issues.  DATB/SCSB can then figure out what the time commitment might be from our side.  I have requested that Sean post a summary/current information on CALSTIS in an issue on HSTCAL repository.
